pykern/pkcli/ci.py

```diff
@@ -8,25 +8,56 @@
 :copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
-from pykern.pkdebug import pkdp, pkdlog
-from pykern import pkunit
-from pykern.pkcli import test
+from pykern import pkcli
+from pykern import pkio
 from pykern import pksetup
+from pykern import pkunit
 import re
 
-
-_FILE_TYPE = re.compile(r".py$")
-_EXCLUDE_FILES = re.compile(
+_CHECK_EOF_NEWLINE_EXCLUDE_FILES = re.compile(
+    r"/ext/|node_modules/|^run/|tests/[\w\d/]+_work/|^venv/"
```

Comment:
This is very specific to sirepo. Also, we do want to skip test data like check_prints_exclude_files does.

Reply:
What's the harm in having these here for any repo? Do you foresee us running this on another repo in the future where we would want to check for newlines in some of these directories?

Reply:
We are going to run this on all repos, eventually. The scientists want to implement CI. It's running on rslinac now.

/ext/ is the only one that's problematic. If it was more qualified, e.g. /static/js/ext/, I'd be less concerned.

Reply:
951542b61ce1c38b913cce136408231f0994e067